src/constants.js

```diff
@@ -1,35 +1,72 @@
 export const BASE_URL =
 	'https://react-fundamentals-course-hw-backend.onrender.com';
```

Comment:
Recomendation (no course scope):
I think I missed this last time.
Basically it's not correct to hardcode endpoints.
Standard approach is to operate with ENV (environment) variables.
So localhost front calls localhost back, dev => dev, prod => prod... etc.
But if you want to target localhost on other envs there should be different starting command in package.json that would setup and ENV variable (LOCAL, DEV, STAGE, PROD) etc.
start:local, start:development etc

Reply:
Added using of environment variables